File: array/2006_count_number_of_pairs_with_absolute_difference_k/2006_count_number_of_pairs_with_absolute_difference_k.go
package main

func countKDifference(nums []int, k int) int {
	result := 0
	for i := 0; i < len(nums); i++ {
		for j := i; j < len(nums); j++ {
			if abs(nums[i]-nums[j]) == k {
				result += 1
			}
		}
	}
	return result
}

func abs(a int) int {
	if a < 0 {
		return -a
	}
	return a
}

File: array/2006_count_number_of_pairs_with_absolute_difference_k/2006_count_number_of_pairs_with_absolute_difference_k.py
class Solution:
    def countKDifference(self, nums: List[int], k: int) -> int:
        result = 0
        for i in range(len(nums)):
            for j in range(i+1, len(nums)):
                if abs(nums[i]-nums[j]) == k:
                    result += 1
        return result
